src/services/test-helpers/responses/blocks/blocks789629.json

```diff
@@ -52,11 +52,34 @@
               "class": "Mandatory",
               "paysFee": "Yes"
             }
+          ],
+          "docs": [
+            " An extrinsic completed successfully. [info]"
           ]
         }
       ],
       "success": true,
-      "paysFee": true
+      "paysFee": true,
```

Comment:
what's up with this file?

Reply:
Is used to test the response from the service. This way anytime there is a user contract change in the service it forces one to change this file and document the change. Additionally it helps prevent accidental regressions in the user contract.